kubectl/kubectl.go

```diff
@@ -0,0 +1,44 @@
+package kubectl
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os/exec"
+	"strings"
+)
+
+// Path is the path of the kubectl binary
+var Path = "kubectl"
+
+// Runner is an interface to wrap kubectl convenience methods
+type Runner interface {
+	ClusterInfo() ([]byte, error)
```

Comment:
can you put some godocs for each of these interface functions? if possible, it would be nice to run [`golint`](https://github.com/golang/lint) on the `kubectl` package, as it will tell you the major points for improvement